### PR TITLE
Add tests for reading and setting variables.

### PR DESCRIPTION
**Pull request by jcrwilliams**
Adds a test suite for reading and setting variables in a simple C
program.

Ref #15 

Change-Id: If16867f4e358ca58cedde31c629cd8159842f28e
Signed-off-by: Jonathan Williams <jonwilliams@blackberry.com>

### CODE REVIEW

File: src/GDBDebugSession.ts [hunk 1]
```diff
@@ -313,7 +313,6 @@ export class GDBDebugSession extends LoggingDebugSession {
                 new Scope('Local', this.variableHandles.create(frame), false),
             ],
         };
-
```
Comment (simark) [inline src/GDBDebugSession.ts]: Unrelated change.

File: src/integration-tests/var.spec.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
+
```
Comment (simark) [inline src/integration-tests/var.spec.ts]: License header.

**Comment by simark** [comment]
This LGTM with the little nits fixed.

File: src/integration-tests/test-programs/vars.c [hunk 3]
```diff
@@ -0,0 +1,7 @@
+int main()
```
Comment (simark) [inline src/integration-tests/test-programs/vars.c]: Do we need license header in these small test programs too?

Comment (jcrwilliams) [reply src/integration-tests/test-programs/vars.c]: I don't think a program like this is copyrightable given its simplicity

File: src/GDBDebugSession.ts (hunk 1 above)
Comment (simark) [reply src/GDBDebugSession.ts]: This hunk is not related to your change, can you please remove it?

**Comment by simark** [comment]
Oh, can you format your documents according to the `.editorconfig` file?  That means 4 spaces per indent for typescript and C files.

**Comment by jonahgraham** [comment]
This build is hanging: https://ci.eclipse.org/cdt/job/cdt-gdb-adapter-verify/41/console 

I am cancelling it and putting a timelimit on the build time.

**Comment by jonahgraham** [comment]
Sorry, last comment was on wrong PR. Correct comment on https://github.com/eclipse-cdt/cdt-gdb-adapter/pull/34#issuecomment-443868066
